.github/workflows/codeql.yml

```diff
@@ -26,7 +26,6 @@ on:
 
 jobs:
   analyze:
-    if: github.event.pull_request.draft == false
```

Comment:
this was a bug (duped key), I dropped it as it was spamming my email :D

Reply:
Good call